package-testing/ruby-sdk-relay/docker-run.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
it may be easier to use build-and-run.sh instead of wrapping in a docker container. PHP was probably not the best example of how to set up the run scripts.
I think you'll want to use [build-and-run.sh](https://github.com/Eppo-exp/sdk-test-data/blob/main/package-testing/sdk-test-runner/README.md#build-and-runsh) and switch on the platform variable (`EPPO_SDK_PLATFORM `) to handle build specifics for each (windows, macos, ubuntu)
`build-and-run.sh` will also need to install gems at the version specified by `SDK_VERSION` or github reference specified by `SDK_REF`

Reply:
I realized ruby is not installed natively!